src/components/RightSidebar/Matterbridge/BridgePart.vue

```diff
@@ -20,26 +20,47 @@
 -->
 
 <template>
-	<div class="part">
+	<div :class="{ part: true, readonly: !editing }">
 		<h3>
+			<img class="icon-service"
+				:src="type.iconUrl" />
 			<span>
 				{{ type.name }}
 			</span>
-			<a v-if="type.infoTarget"
-				class="icon icon-info"
-				target="_blank"
-				:href="type.infoTarget" />
-			<button v-if="deletable"
-				class="icon icon-delete"
-				@click="$emit('deletePart')" />
+			<Actions
+				:force-menu="false">
+				<ActionButton
+					:icon="editing ? 'icon-checkmark' : 'icon-rename'"
+					@click="onEditClick">
+					{{ editing ? t('spreed', 'Ok'): t('spreed', 'Edit') }}
```

Comment:
"Ok" is not an action. what do you mean? Save?

Reply:
Well this button does not really save, it just validates the edition. Would "Validate" be ok?

Reply:
Check?
Or just do it automatically after a timeout (so you can use `debounce()`)

Reply:
What about allowing edition only if the bridge is disabled and save each time a part is validated? So we can call this edition validation button "save" and get rid of the big general "save" button.

Reply:
Sounds like it would help to get rid of confusion and help to avoid misbehaviour